packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -749,4 +749,32 @@
         <protected>false</protected>
         <shortDescription>Timeframe facet end label</shortDescription>
     </labels>
+    <labels>
```

Comment:
@acarpentier01  please tell me if everything looks fine to you in the following labels.

Reply:
Absolutely! Thank you for your feedback👌